Update Contributing section and credits (#124)
VSpaceCode/VSpaceCode: README.md

See [CHANGELOG.md](CHANGELOG.md)

## Contributing

All feature requests and help are welcome. Please check out our [contributing guide](CONTRIBUTING.md) on how you can help.

> This project is maintained by a [group of people](https://github.com/VSpaceCode/VSpaceCode/graphs/contributors) around the world on their free time. Thank you for your contribution.❤️

Reviewer: ```suggestion
This project is maintained by different [people](https://github.com/VSpaceCode/VSpaceCode/graphs/contributors) around the world in their free time. Thank you for your contribution. ❤️
```
